data/tlg2042/tlg053/tlg2042.tlg053.1st1K-grc1.xml

```diff
@@ -0,0 +1,360 @@
+                    <resp>Editor-in-Chief, Perseus Digital Library</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Matt Munson</persName>
+                    <resp>Project Manager (University of Leipzig), 2016 - present</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Annette Gessner</persName>
+                    <resp>Project Assistant (University of Leipzig) 2015 - present</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Thibault Clérice</persName>
+                    <resp>Lead Developer (University of Leipzig)</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Bruce Robertson</persName>
+                    <resp>Technical Advisor</resp>
+                </respStmt>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Harvard College Library</authority>
+                <idno type="filename">tlg2042.tlg053.1st1K-grc1.xml</idno>
+                <availability>
+                    <p>Available under a Creative Commons Attribution-ShareAlike 4.0 International
+                        License</p>
+                </availability>
+                <date>2017</date>
+                <publisher>Harvard College Library</publisher>
+                <pubPlace>United State</pubPlace>
+            </publicationStmt>
+            <sourceDesc>
+                <listBibl>
+                    <biblStruct>
+                        <monogr>
+                            <title xml:lang="lat">Origenis Opera Omnia</title>
+                            <editor>
+                                <persName>
+                                    <name>Charles Delarue</name>
+                                </persName>
+                            </editor>
+                            <author ref="urn:cts:greekLit:tlg2042">Origen</author>
+                            <imprint>
+                                <publisher>J. P. Migne</publisher>
+                                <pubPlace>Paris</pubPlace>
+                                <date>1857</date>
+                            </imprint>
+                        </monogr>
+                        <series>
+                            <title level="s" xml:lang="lat">Patrologiae cursus completus (series
+                                Graeca)</title>
+                            <biblScope unit="volume">12</biblScope>
+                        </series>
+                        <ref
+                            target="https://archive.org/stream/patrologiae_cursus_completus_gr_vol_012#page/n407/mode/2up"
+                            >Internet Archive</ref>
+                    </biblStruct>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>Text encoded in accordance with the latest EpiDoc standards (January 2014)</p>
```

Comment:
I haven't been checking this in your other PRs but the EpiDoc standards that we are using are no longer those from January 2014. Since we will always update the RNG file when there is a new one, I think we can completely remove the date from this, so that the text only reads "Text encoded in accordance with the latest EpiDoc standards".